scarb/src/bin/scarb/commands/fmt.rs

```diff
@@ -7,13 +7,15 @@ use scarb::ops;
 #[tracing::instrument(skip_all, level = "info")]
 pub fn run(args: FmtArgs, config: &Config) -> Result<()> {
     let ws = ops::read_workspace(config.manifest_path(), config)?;
-    ops::format(
+    match ops::format(
         ops::FmtOptions {
             check: args.check,
-            all: args.all,
             pkg_name: args.package,
             color: !args.no_color,
         },
         &ws,
-    )
+    ) {
+        Ok(true) => Ok(()),
+        _ => std::process::exit(1),
+    }
```

Comment:
Move exit code handling to error types system, basically introduce `ErrorWithExitCode` struct using `thiserror` or smth like this. See `CargoResult` in Cargo. This is to be done in separate PR.

Reply:
https://github.com/software-mansion/scarb/issues/70